WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -178,8 +178,19 @@ class AztecPostViewController: UIViewController {
         }
     }
 
+
+    /// Active Downloads
+    ///
     fileprivate var activeMediaRequests = [AFImageDownloadReceipt]()
 
+
+    /// Boolean indicating whether the post should be removed whenever the changes are discarded, or not.
+    ///
+    fileprivate var shouldRemovePostOnDiscard = false
```

Comment:
Consider renaming to `removePostOnDiscard` since once this is set, there should be no doubts about what will happen.